src/native/libs/System.Native/pal_time.c

```diff
@@ -14,6 +14,9 @@
 #include <sys/resource.h>
 #if HAVE_CLOCK_GETTIME_NSEC_NP
 #include <time.h>
+#include <stio.h>
+#include <typeinfo>
+#include <iostream>
```

Comment:
These headers shouldn't be necessary.

```suggestion
```

Reply:
I do not see this feedback addressed. These unnecessary includes should be deleted.